ansible_collections/arista/avd/examples/campus-fabric/intended/configs/SPINE1.cfg

```diff
@@ -0,0 +1,239 @@
+!RANCID-CONTENT-TYPE: arista
```

Comment:
maybe worth adding uplinks to core/wan to the example?

Reply:
yes I will do that.

Reply:
Something like this?

![image](https://user-images.githubusercontent.com/25393619/197839339-5cbeca04-7a0d-4138-a3f4-3f6c805e08d5.png)

Reply:
Something like this?

![image](https://user-images.githubusercontent.com/25393619/197853528-444a3137-09fd-4810-ac57-00ef638fd9ad.png)

Reply:
landed with this version...

![image](https://user-images.githubusercontent.com/25393619/198017682-858c68bf-a53d-4aa8-bd95-6e4573caffa0.png)